src/helpers/iq.ts

```diff
@@ -18,5 +18,12 @@ import { IqWebhookPayloadApplicationEvaluation } from "../types";
 import { IQ_SERVER_URL } from '..';
 
 export function getIqUrlForApplicationEvaluation(payload: IqWebhookPayloadApplicationEvaluation): string {
```

Comment:
I would normalize the IQ SERVER URL in `index.ts` rather than in here (revert this change).

Reply:
Could be written on one line:
`const IQ_SERVER_URL = (IQ_SERVER_URL.slice(-1) == "/" ? IQ_SERVER_URL : `${IQ_SERVER_URL}/`)`

Reply:
...you did this to me...

export const IQ_SERVER_URL: string = (process.env.IQ_SERVER_URL.slice(-1) != "/") ? process.env.IQ_SERVER_URL+"/" : process.env.IQ_SERVER_URL